src/test/storm/modelchecker/prctl/mdp/MdpPrctlModelCheckerTest.cpp

```diff
@@ -245,6 +245,7 @@ class SparseDoubleLPEnvironment {
         storm::Environment env;
         env.solver().minMax().setMethod(storm::solver::MinMaxMethod::LinearProgramming);
         env.solver().minMax().setPrecision(storm::utility::convertNumber<storm::RationalNumber>(1e-6));
+        env.solver().minMax().setRelativeTerminationCriterion(false);
```

Comment:
This seems unrelated to the given PR.
```suggestion
```

Reply:
It is necessary for result to be computed correctly with the configuration. Alternatively I can configure the test to be skipped for configs without the absolute termination criterion set